README.md

```diff
@@ -32,71 +30,122 @@ helm install forklift-console-plugin forklift/forklift-console-plugin \
   --set forklift.namespace=<the forklift operator namespace>
 ```
 
-For more information about forklift-console-plugin deployment, see the [forklift-console-plugin docs](https://github.com/kubev2v/forklift-console-plugin/tree/main/docs).
+forklift-console-plugin will add new menu item under the `virtualization` secition for virtualization workloads import in [Openshift web console](https://github.com/openshift/console).
+
+## Development
+
+### Using the development constainer image
+
+Use an Openshift cluster you have admin privileges on, or spin your own [OpenShift Local](https://developers.redhat.com/products/openshift-local) cluster, Install Forklift with console plugin feature turned off.
+On your cluster, grant privilided capabilites to the development service acount, and deploy the development enviorment,
+if you do not have admin access to an Openshift cluster, you can use [KinD](https://sigs.k8s.io/kind) or [OpenShift Local](https://developers.redhat.com/products/openshift-local) to create a small development cluster on your PC.
+
+``` bash
+# deploy the development pod
+NAMESPACE=openshift-mtv
+IMAGE=quay.io/kubev2v/forklift-console-plugin-dev
+TEMPLATE=https://raw.githubusercontent.com/kubev2v/forklift-console-plugin/main/scripts/yaml/forklift-plugin-dev.tpl.yaml
```

Comment:
Note when testing this, this template does not exist yet, it will be added by this PR